src/pages/_playground/qa/[content-type]/review.tsx

```diff
@@ -18,14 +18,58 @@ interface ReviewStatus {
   [path: string]: boolean
 }
 
+interface ChecklistStatus {
+  [path: string]: Record<string, boolean>
+}
+
 interface ReviewRowProps {
   qaPath: QAPath
   reviewed: boolean
   onReviewToggle: (path: string, reviewed: boolean) => void
+  checklistState: Record<string, boolean>
+  onChecklistToggle: (path: string, itemId: string) => void
+}
+
+const QA_CHECKLIST = {
```

Comment:
I didn't spend a ton of time on these items, so they're very open to suggestions.